rest-api/api.ini

```diff
@@ -0,0 +1,11 @@
+[Director]
+Name=bareos-dir
+Address=127.0.0.1
+Port=9101
+
+[JWT]
+# to get a string like this run:
+# openssl rand -hex 32
+secret_key = 936959a2a6902056b924669796c74aad13b9da2b5cf637b70e377b3d7c29c6fb
```

Comment:
it is not a random value, so we have to generate the key using openssl. I remove the default key and it will stop on startup, if no key is configured

Reply:
Couldn't you just generate a value on startup if this isn't configured? This would be great for testing, because you don't need to configure it. In production - where you want a client's JWT to be valid after a restart - this would have to be configured, of course.